### PR TITLE
 models, controllers, services and dao visualization team

### PR DESCRIPTION
**Pull request by SinghNikhil1**

### CODE REVIEW

File: CurriculaService/src/main/java/com/revature/repositories/VisualizationDao.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.revature.models.Visualization;
+
+
+
+
+
+public interface VisualizationDao  extends JpaRepository<Visualization, Integer> {
+	public Visualization findVisualizationById(int id);
```
Comment (Abatson) [inline CurriculaService/src/main/java/com/revature/repositories/VisualizationDao.java]: This function is already provided by default in the JpaRepository, as getOne() or findById()

**Review by Abatson — CHANGES_REQUESTED** [review]
Review VisualizationDao

File: CurriculaService/src/main/java/com/revature/repositories/VisualizationDao.java (hunk 1 above)
Comment (SinghNikhil1) [reply CurriculaService/src/main/java/com/revature/repositories/VisualizationDao.java]: I deleted line 12

File: CurriculaService/src/main/java/com/revature/services/VisualizationServiceImpl.java [hunk 2]
```diff
@@ -19,7 +19,7 @@ public VisualizationServiceImpl(VisualizationDao vd) {
 	@Override
 	public Visualization findVisualizationById(int id) {
 		// TODO Auto-generated method stub
-		return vd.findVisualizationById(id);
+		return vd.getOne(id);
 	}
```
Comment (SinghNikhil1) [inline CurriculaService/src/main/java/com/revature/services/VisualizationServiceImpl.java:23]: rename the function

**Review by SinghNikhil1 — COMMENTED** [review]
this is my changes

File: CurriculaService/src/main/java/com/revature/services/VisualizationServiceImpl.java (hunk 2 above)
Comment (Abatson) [reply CurriculaService/src/main/java/com/revature/services/VisualizationServiceImpl.java:23]: Just delete it Nikhil

**Review by Abatson — APPROVED** [review]

**Pull request merged**